### PR TITLE
add the Admin donate page 

### PR DESCRIPTION
**Pull request by AbdallahAmmar96**
add all relative files to admin donate page

### CODE REVIEW

**Review by InassTubail — CHANGES_REQUESTED** [review]
Remove comments from /controllers/admin/donate/donate.js

File: src/controllers/admin/donate/donate.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const { join } = require('path');
+const donates = require('../../../database/queries/admin/getQuery');
+// const update = require('');
```
Comment (Jamalat-shamallakh) [inline src/controllers/admin/donate/donate.js:3]: delete comments

**Review by Jamalat-shamallakh — CHANGES_REQUESTED** [review]

File: src/views/admin/donates.hbs [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<main class="main">
+  <section class="taps">
+    <input type="button" value="New Donations" class="new">
+    <input type="button" value="Current Donations" class="current">
+    <input type="button" value="Done Donations" class="done">
+  </section>
+  {{#each card}}
+  {{#unless card.state}}
```
Comment (tomduggan85) [inline src/views/admin/donates.hbs:8]: @AbdallahAmmar96 You might need `this.state` to access the state of the current card in the `#each` loop

File: src/views/admin/donates.hbs [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      <input type="button" value="Personal Info" class="tap--personal">
+      <input type="button" value="Donation" class="tap--donation">
+    </div>
+    {{!-- {{#if !card.state}} --}}
+    <div class="cards--info">
+      <div class="cards--info-personal">
+        <h4 class="fullName">Full Name: {{fullName}}</h4>
+        <h4 class="phoneNumber">Phone Number: {{phoneNumber}}</h4>
+        <h4 class="email">Email: {{email}}</h4>
+        <h4 class="city">City: {{cityName}}</h4>
```
Comment (tomduggan85) [inline src/views/admin/donates.hbs:20]: @AbdallahAmmar96 Are fullName, phoneNumber, email and cityName attributes of the current card in the `#each` loop?  I think then you'd need to use `this.fullName`, `this.phoneNumber`, and so on.
Or it looks like handlebars 3.0 has a feature called block parameters which makes it easier to write `#each` loops, if you're interested:  https://handlebarsjs.com/block_helpers.html#block-params

File: src/views/admin/donates.hbs [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        {{!-- <h4 class="steet">Steet{{streetName}}:</h4> --}}
+      </div>
+      <div class="cards--info-donation hide">
+        <h4 class="piecesNumber">Number Of Pieces: {{numberOfPieces}}</h4>
+        <h4 class="gender">Quality(s): {{quality}}</h4>
+        <h4 class="quality">Gender(s): {{gender}}</h4>
+        <h4 class="season">Season(s): {{season}}</h4>
```
Comment (tomduggan85) [inline src/views/admin/donates.hbs:27]: @AbdallahAmmar96 Similar to my above comment, you might need `this.numberOfPieces`, `this.quality`, and so on.

**Pull request closed without merging**